server/app.js

```diff
@@ -1,17 +1,13 @@
 import express from 'express';
```

Comment:
models/index.js is a file generated from sequelize-cli, it has been converted to ES6 while still using require rather than import. Its a hands off file that shouldn't be touched anyway. 

Any other files generated from sequelize-cli are in ES5 format due to a lack of ES6 support. This means only be the generated model itself, and the migrations will be ES5, everything else is in ES6.

Minor issue, could probably be ignored IMO.